internal/app/machined/pkg/controllers/k8s/manifest.go

```diff
@@ -82,6 +82,7 @@ func (ctrl *ManifestController) Run(ctx context.Context, r controller.Runtime, l
 			return err
 		}
 
+		configVersion := configResource.(*config.K8sControlPlane).Metadata().Version().String()
```

Comment:
this won't quite work, as Talos now doesn't ever _update_ resources from manifests, it only creates them if they're missing

Reply:
ok, I'll remove that.

So keep only replication count parameter?